doc/api/sqlite.md

```diff
@@ -322,11 +322,15 @@ over hand-crafted SQL strings when handling user input.
 
 <!-- YAML
 added: v22.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56385
+    description: support for TypedArray and DataView.
```

Comment:
nit: We can use the same wording as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/isView
```suggestion
    description: Add support for `DataView` and typed array objects for `anonymousParameters`.
```